package.json

```
@@ -94,5 +94,8 @@
     "after": [
       "ember-changeset"
     ]
+  },
+  "peerDependencies": {
+    "@ember/string": "^4.0.0"
```

@jelhan can we keep this
```suggestion
    "@ember/string": "^3.0.0 || ^4.0.0"
```
or
```suggestion
    "@ember/string": ">=3.0.0"
```

to simplify community migration? e.g. ember-data v4.12 depends on `@ember/string` v3